test/es-module/test-esm-tla-unfinished.mjs

```diff
@@ -72,24 +72,31 @@ describe('ESM: unsettled and rejected promises', { concurrency: !process.env.TES
   });
 
   it('should exit for an unsettled TLA promise with warning', async () => {
-    const { code, stderr, stdout } = await spawnPromisified(execPath, [
+    const { code, stderr } = await spawnPromisified(execPath, [
```

Comment:
Why are you removing `stdout` checks? It's quite useful to get when something's wrong to get the output

Reply:
The `stdout` check here has been removed because the output of the `'es-modules/tla/unresolved.mjs'` is being tested in the next test, so everything is still being fully tested

Reply:
Everything is also tested if you add it back, which would be my preference